scripts/gsa-approved-software.js

```diff
@@ -14,24 +14,30 @@ const csv = require('csvtojson');
 let _robot = null;
 
 const colors = {
-  'Proposed': '#f9c642',
-  'Approved': '#2e8540',
-  'Sunsetting': '#e7f4e4',
-  'Denied': '#e31c3d'
+  Proposed: '#f9c642',
```

Comment:
A lot of this file is just changed by my code formatter.  It's basically using [prettier](https://www.npmjs.com/package/prettier) without any extra configuration, I think.

Reply:
Did you check the actual hex codes to ensure accessibility? (high enough contrast?)

Reply:
They're USWDS colors.  😄